pkgs/development/python-modules/pythran/default.nix

```diff
@@ -51,9 +57,9 @@ buildPythonPackage rec {
     ln -s '${lib.getDev xsimd}'/include/xsimd pythran/
   '';
 
-  nativeBuildInputs = [ setuptools ];
+  build-system = [ setuptools ];
 
-  propagatedBuildInputs = [
```

Comment:
Thanks for these changes, I'd like semantic changes to appear in a separate commit.